unity-ggjj/Assets/Scripts/NarrativeScript/NarrativeScriptPlayer/INarrativeScriptPlayer.cs

```diff
@@ -1,12 +1,14 @@
 public interface INarrativeScriptPlayer
 {
-    bool Waiting { get; set; }
-    GameMode GameMode { get; set; }
-    bool CanPressWitness { get; }
-    INarrativeScript ActiveNarrativeScript { get; }
-
-    void Continue();
-    void StartSubStory(NarrativeScript gameOverScript);
-    void HandleChoice(int choiceIndex);
-    void PresentEvidence(ICourtRecordObject evidence);
-}
+    public INarrativeScript ActiveNarrativeScript { get; }
+    public GameMode GameMode { get; set; }
+    public bool Waiting { get; set; }
+    public bool CanPressWitness { get; }
+    public bool HasSubStory { get; }
+    
+    public void Continue(bool overridePrintingText = false);
+    public void HandleChoice(int choiceIndex);
+    public void PresentEvidence(ICourtRecordObject courtRecordObject);
+    public void StartSubStory(NarrativeScript narrativeScript);
+    public void SetWaitingToFalseAndContinue();
```

Comment:
Nit: _Superfluous `public`-modifier 2: Electric Boogaloo_
```suggestion
    INarrativeScript ActiveNarrativeScript { get; }
    GameMode GameMode { get; set; }
    bool Waiting { get; set; }
    bool CanPressWitness { get; }
    bool HasSubStory { get; }
    
    void Continue(bool overridePrintingText = false);
    void HandleChoice(int choiceIndex);
    void PresentEvidence(ICourtRecordObject courtRecordObject);
    void StartSubStory(NarrativeScript narrativeScript);
    void SetWaitingToFalseAndContinue();
```

Reply:
One day I'll remember to remove these when copy pasting methods 👍